### PR TITLE
update packages/graphics 

### PR DESCRIPTION
**Pull request by heitbaum**
Graphics packages updated. Mostly minor.

Notable changes:
- libheif had a few version jumps.
- libde265 now using release number not a sha256
- glmark2 now compiled with meson

Notes/Todos: (these packages have not been updated in this PR)
- looks like **vsxu** has not been updated for a while? It has some updated releases, the GitHub is archived. Do we want to update this? 
- kmscube - skipping only a single commit
- cairo - are we bumping to 1.17.4? Also note maintainer update - https://gitlab.freedesktop.org/cairo/cairo/-/blob/156cd3eaaebfd8635517c2baf61fcf3627ff7ec2/NEWS
- glfw - working through the Xcursor dependancy with glfw-3.3.2
 

```
=== tested on ===
 01:00:25 up 1 day,  2:31,  load average: 1.27, 1.22, 1.25
Generic.x86_64-devel-20201222112050-a9a38e4
Linux LibreELEC 5.10.2 #1 SMP Mon Dec 21 14:09:09 UTC 2020 x86_64 GNU/Linux
Starting Kodi (19.0-BETA2 (18.9.821) Git:19.0b2-Matrix). Platform: Linux x86 64-bit
Using Release Kodi x64
Kodi compiled 2020-12-15 by GCC 10.2.0 for Linux x86 64-bit version 5.10.0 (330240)
Running on LibreELEC (community): devel-20201222112050-a9a38e4 9.80, kernel: Linux x86 64-bit version 5.10.2
FFmpeg version/source: 4.3.1-Kodi
Host CPU: Intel(R) Core(TM) i5-6260U CPU @ 1.80GHz, 4 cores available
[    0.000000] DMI:  /NUC6i5SYB, BIOS SYSKLi35.86A.0073.2020.0909.1625 09/09/2020
```

### CODE REVIEW

**Comment by heitbaum** [comment]
Rebase against #4803

File: packages/graphics/glmark2/package.mk [hunk 1]
```diff
@@ -2,31 +2,19 @@
...
-PKG_SHA256="c0fb0fb2f62d05f229466a4c1cbb902816595b0a5ed860de46f763533c515e6c"
+PKG_VERSION="784aca755a469b144acf3cae180b6e613b7b057a"
+PKG_SHA256="787dd7de05c795c7c42694e648b5c6418e4b395fc460ea349d1d35d493469356"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/glmark2/glmark2"
 PKG_URL="https://github.com/glmark2/glmark2/archive/$PKG_VERSION.tar.gz"
 PKG_DEPENDS_TARGET="toolchain"
 PKG_LONGDESC="glmark2 is an OpenGL 2.0 and ES 2.0 benchmark"
-PKG_TOOLCHAIN="manual"
+PKG_TOOLCHAIN="meson"
```
Comment (lrusak) [inline packages/graphics/glmark2/package.mk:12]: should be able to drop this line completely

File: packages/graphics/lcms2/package.mk [hunk 2]
```diff
@@ -2,10 +2,11 @@
...
 PKG_NAME="lcms2"
-PKG_VERSION="2.9"
-PKG_SHA256="8e23a09dc81af856db37941a4ea26acdf6a45b0281ec5b7ee94b5a4e9f7afbf7"
-PKG_LICENSE="MIT"
+PKG_VERSION="2.11"
+PKG_SHA256="dc49b9c8e4d7cdff376040571a722902b682a795bf92985a85b48854c270772e"
+PKG_LICENSE="MIT/GPLv3"
 PKG_SITE="http://www.littlecms.com"
-PKG_URL="https://github.com/mm2/Little-CMS/archive/lcms${PKG_VERSION}.tar.gz"
+PKG_URL="$SOURCEFORGE_SRC/lcms/lcms/${PKG_VERSION}/${PKG_NAME}-${PKG_VERSION}.tar.gz"
```
Comment (lrusak) [inline packages/graphics/lcms2/package.mk:9]: why change to sourceforge?

File: packages/graphics/lcms2/package.mk [hunk 3]
```diff
@@ -2,10 +2,11 @@
...
 PKG_DEPENDS_TARGET="toolchain tiff"
 PKG_LONGDESC="An small-footprint color management engine, with special focus on accuracy and performance."
+PKG_TOOLCHAIN="configure"
```
Comment (lrusak) [inline packages/graphics/lcms2/package.mk:12]: Is this actually needed?

File: packages/graphics/libepoxy/package.mk [hunk 4]
```diff
@@ -8,11 +8,12 @@
...
 
 PKG_NAME="libepoxy"
-PKG_VERSION="1.5.4"
-PKG_SHA256="0bd2cc681dfeffdef739cb29913f8c3caa47a88a451fd2bc6e606c02997289d2"
+PKG_VERSION="1.5.5"
+PKG_SHA256="5d80a43a6524a1ebdd0c9c5d5105295546a0794681853c636a0c70f8f9c658ce"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/anholt/libepoxy"
-PKG_URL="https://github.com/anholt/libepoxy/releases/download/${PKG_VERSION}/libepoxy-${PKG_VERSION}.tar.xz"
+#PKG_URL="https://github.com/anholt/libepoxy/releases/download/${PKG_VERSION}/libepoxy-${PKG_VERSION}.tar.xz"
```
Comment (lrusak) [inline packages/graphics/libepoxy/package.mk:15]: what's the reason for not using the release url?

**Review by lrusak — CHANGES_REQUESTED** [review]

File: packages/graphics/lcms2/package.mk (hunk 3 above)
Comment (heitbaum) [reply packages/graphics/lcms2/package.mk:12]: Checked - not required 
```
$ PROJECT=Generic ARCH=x86_64 IMAGE_SUFFIX=rudi s/build lcms2
UNPACK      lcms2
    FIXCONFIG      /home/rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-9.80-devel/build/lcms2-2.11/
BUILD      lcms2 (target)
    TOOLCHAIN      configure (auto-detect)
Executing (target): /home/rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-9.80-devel/build/lcms2-2.11/configure --host=x86_64-libreelec-linux-gnu --build=x86_64-linux-gnu --prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --libdir=/usr/lib --libexecdir=/usr/lib --localstatedir=/var --disable-static --enable-shared 
````

File: packages/graphics/glmark2/package.mk (hunk 1 above)
Comment (heitbaum) [reply packages/graphics/glmark2/package.mk:12]: 👍
```
$ PROJECT=Generic ARCH=x86_64 IMAGE_SUFFIX=rudi s/build glmark2
UNPACK      glmark2
BUILD      glmark2 (target)
    TOOLCHAIN      meson (auto-detect)
Executing (target): meson --prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --libdir=/usr/lib --libexecdir=/usr/lib --localstatedir=/var --buildtype=plain -Dstrip=true --cross-file=/home/rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-9.80-devel/build/glmark2-784aca755a469b144acf3cae180b6e613b7b057a/.x86_64-libreelec-linux-gnu/meson.conf -Dflavors=drm-gl /home/rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-9.80-devel/build/glmark2-784aca755a469b144acf3cae180b6e613b7b057a
```

File: packages/graphics/libepoxy/package.mk (hunk 4 above)
Comment (heitbaum) [reply packages/graphics/libepoxy/package.mk:15]: Don’t think it was working or that I can’t recall 🤔 
Fixed any ways 😀

File: packages/graphics/lcms2/package.mk (hunk 2 above)
Comment (heitbaum) [reply packages/graphics/lcms2/package.mk:9]: @mm2 moved the release to SF. https://github.com/mm2/Little-CMS/releases/tag/2.11

Comment (lrusak) [reply packages/graphics/lcms2/package.mk:9]: meh, if it works I guess it's fine

**Review by lrusak — APPROVED** [review]
LGTM

**Pull request merged**